doc/api/packages.md

```diff
@@ -335,18 +356,17 @@ with the main entry point by treating the main entry point as the
 
 ```json
 {
-  "main": "./main.js",
   "exports": {
-    ".": "./main.js",
-    "./submodule": "./src/submodule.js"
+    ".": "./index.js",
+    "./submodule.js": "./src/submodule.js"
```

Comment:
```suggestion
    "./submodule": "./src/submodule.js"
```

Reply:
i still think these examples shouldn't be changed to _add_ the extension.

Reply:
I don’t really understand what’s motivating this change either, if anything that adds more bloat to the import maps – I mean it adds 3 bytes; jokes aside, it could give the reader the impression that using an extension is required when defining an export path, and/or confuse needlessly the package consumers on why the is no `submodule.js` file at the trot of the package – whereas extensionless makes a good job signaling we’re not dealing with actual paths imo. 
(Same thing for the change `main.js` -> `index.js`, I don’t think it’s necessary)

That being said, I don’t feel strongly about this, so feel free to disregard.